lib/mapper.js

```diff
@@ -22,13 +52,52 @@ exports.track = function(track){
     objCase.del(props, 'revenue');
   }
 
-  return {
-    email: track.email(),
-    userId: track.userId(),
-    eventName: track.event(),
-    createdAt: unixTime(track.timestamp()),
-    dataFields: props
-  };
+  var eventName = track.event();
+  var cart = props["products"] || props["cart"];
```

Comment:
As part of this PR, we want to support adding an (as I understand non-standard) "products" or "cart" object to the Added/Removed Product calls. How do I correctly look check ```cart``` if ```track.products()``` isn't set?